src/components/resource-title/ResourceTitle.scss

```diff
@@ -2,13 +2,14 @@
 
 .resource-title {
   align-items: center;
-  border-bottom: 1px solid var(--hauki-content-border-color);
-  box-shadow: 0 2px 10px 0 rgba(212, 212, 212, 0.2);
+  background: #fff;
   display: flex;
-  gap: 1.5rem;
   justify-content: space-between;
   left: 0;
+  margin-left: -1rem;
```

Comment:
Should we use HDS sass-variables here as well?

Reply:
Lets do it.